src/Pages/Test.tsx

```diff
@@ -1,7 +1,15 @@
-import { ReactElement } from "react";
+import { ReactElement, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { RootState } from "../redux/Store";
+import { fetchCourses } from "../redux/actions/CoursesActions";
 
-const Test = (): ReactElement => {
-  return <div>Test</div>;
-};
+export const Test = (): ReactElement => {
+  const course = useSelector((state: RootState) => state.courseslist.value);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    !course.length && dispatch(fetchCourses());
+  }, []);
 
-export default Test;
+  return <div>{course && course.map((e) => <div key={e.id}>{e.id}</div>)}</div>;
```

Comment:
Here should be not a list.... but course question. For now you can just get from store, by ID (passed in query selector) and print here course name.